manifests/sensor.yaml

```diff
@@ -0,0 +1,13 @@
+name: sensor
+authors: ORBIT
+homepage: https://github.com/orbitalsupershell/sensor/
+license: ALL RIGHTS RESERVED
```

Comment:
I notice your plugin copies some assets from the base game and there's no debian copyright file with attribution.

Please add a copyright file with attribution and sources for the images you copied.

Reply:
Working on a newer plugin, consider this PR closed.

Reply:
Please review copyright @samrocketman

Reply:
Looks ok to me.